postgres-appliance/scripts/configure_spilo.py

```diff
@@ -689,39 +697,38 @@ def pystache_render(*args, **kwargs):
 
 
 def get_dcs_config(config, placeholders):
+    # (KUBERNETES|ZOOKEEPER|EXHIBITOR|CONSUL|ETCD3|ETCD)_(HOSTS|HOST|PORT|...)
+    dcs_configs = defaultdict(dict)
+    for name, value in placeholders.items():
+        if '_' not in name:
+            continue
+        dcs, param = name.lower().split('_', 1)
+        if dcs in PATRONI_DCS:
```

Comment:
should not that raise a warning if the condition does not hold ?

Reply:
No, there could be many different environment variables, this is absolutely normal.